autokeras/blocks/basic.py

```diff
@@ -272,13 +274,17 @@ def __init__(self,
                  version: Optional[str] = None,
                  pooling: Optional[str] = None,
                  **kwargs):
+        kwargs.pop('include_top', None)
+        kwargs.pop('input_shape', None)
```

Comment:
If these arguments are not supported, we should raise an error if they are passed, to avoid user confusion.